vault/logical_system.go

```diff
@@ -18,17 +18,17 @@ import (
 	"sync"
 	"time"
 
-	multierror "github.com/hashicorp/go-multierror"
-	"github.com/hashicorp/vault/physical/raft"
-
 	"github.com/hashicorp/errwrap"
 	log "github.com/hashicorp/go-hclog"
 	memdb "github.com/hashicorp/go-memdb"
+	multierror "github.com/hashicorp/go-multierror"
```

Comment:
This doesn't need to specify `multierror` anymore I don't think, as of Go 1.13.

Reply:
I'll try taking it out